sonarts-sq-plugin/sonar-typescript-plugin/src/test/java/org/sonar/plugin/typescript/ContextualServerTest.java

```diff
@@ -98,8 +115,17 @@ public void should_provide_analysis_results() throws Exception {
     assertThat(analyze.cpdTokens).isEmpty();
   }
 
+  @Test
+  public void consume_stdout_stderr() throws Exception {
+    ContextualServer contextualServer = getContextualServer();
+    contextualServer.start();
+    Thread.sleep(2_000);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "Thread.sleep()". ([squid:S2925](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2925))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&branch=issue-443&issues=AWEiyOv91DTsLFfQjB93&open=AWEiyOv91DTsLFfQjB93)

[ISSUE_KEY]: AWEiyOv91DTsLFfQjB93

Reply:
This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED